### PR TITLE
fix: Prevent no donation

### PR DESCRIPTION
**Pull request by anhanh11001**
Details:
- Disable options to pick donation ticket quantity when no donation is entered

Fixes: #2243 

Screenshots for the change:
ezgif-1-6a79ddaac642

### CODE REVIEW

File: app/schemas/org.fossasia.openevent.general.OpenEventDatabase/9.json [hunk 1]
```diff
@@ -2029,6 +2035,18 @@
...
+            "fieldPath": "subTitle",
+            "columnName": "subTitle",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "longAbstract",
+            "columnName": "longAbstract",
+            "affinity": "TEXT",
+            "notNull": false
```
Comment (iamareebjamal) [inline app/schemas/org.fossasia.openevent.general.OpenEventDatabase/9.json]: Why changes in this?

Comment (anhanh11001) [reply app/schemas/org.fossasia.openevent.general.OpenEventDatabase/9.json]: reverted

**Comment by adityastic** [comment]
> Build Failing

Travis was having issues this morning, please rerun the build manually from travis @iamareebjamal

**Comment by anhanh11001** [comment]
@iamareebjamal from what I'm seeing here, it is not??? all checks have passed

**Comment by iamareebjamal** [comment]
It was failing when I commented

**Comment by ShridharGoel** [comment]
> from what I'm seeing here, it is not??? all checks have passed

I had restarted the build, then it passed.

**Pull request merged**